soc/nordic/nrf92/mpu_regions.c

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
This file looks like it could be reused for nrf54h. Would it be possible to share it?